torch/_export/utils.py

```diff
@@ -165,6 +165,8 @@ def _populate_param_buffer_metadata_to_new_gm(
                 param_name = new_sig.inputs_to_parameters[node.target]
                 if param_name in params_buffers_to_node_meta:
                     for k, v in params_buffers_to_node_meta[param_name].items():
+                        if k == "custom":
+                            continue
```

Comment:
@yiming0416 just curious why do we need to skip over this?

Reply:
Because the issue of @jerryzh168 is that the `custom` field is copied to `get_attr` nodes from the `call_function` nodes unexpectedly which causes the quantization test to fail.

I don't understand why we need the copy during the populating process though...

Reply:
as discussed offline with @yiming0416 I think the fix should be in: https://github.com/pytorch/pytorch/blob/main/torch/_export/utils.py#L136-L143

Reply:
Hmm this seems bit hacky, let's discuss offline whether we actually need this logic in the first place.